.github/workflows/lint.yml

```diff
@@ -57,4 +57,4 @@ jobs:
       - uses: stefanzweifel/git-auto-commit-action@v4
         with:
           commit_message: "GPC-NONE: Automatically lint files"
-          file_pattern: '*.kt *.ts *.tf platform/gradle.lockfile *.java'
+          file_pattern: '*.kt *.ts *.tf **/gradle.lockfile *.java'
```

Comment:
Yeah, I think you're right that it was originally because there was only the one. Now that there are more, this glob pattern incorporates them all in one step and I don't think there are currently any that we would want to ignore but definitely something to keep in mind for future

Reply:
Cool, sounds good - nice spot 🙂